crates/holochain/src/core/ribosome/host_fn/send_remote_signal.rs

```diff
@@ -181,7 +180,7 @@ mod tests {
 
         let mut signals = Vec::new();
         for h in conductors.iter() {
-            signals.push(h.signal_broadcaster().subscribe_merged())
```

Comment:
There were multiple ways into this and it was a rather complex streaming join across multiple interfaces. The only test pattern is now to subscribe the way an app does, to a stream of signals from an app.

Reply:
It's still doing an 'internal' subscribe rather than needing a websocket, so you can test without websockets and authentication - but it's otherwise identical to the way an app would get signals.